lib/mutant/isolation.rb

```diff
@@ -1,31 +1,35 @@
 module Mutant
   # Module providing isolation
-  module Isolation
+  class Isolation
     Error = Class.new(RuntimeError)
 
-    module None
-
+    class None < self
       # Call block in isolation
       #
       # @return [Object]
       #
       # @raise [Error]
       #   if block terminates abnormal
-      def self.call(&block)
-        block.call
+      def call
+        yield
       rescue => exception
```

Comment:
Do you want to ignore exceptions that don't inherit from `StandardError`? If you want to catch all exceptions then you probably want `rescue Exception => exception`.